docs/variant_annotation.md

```diff
@@ -0,0 +1,140 @@
+enables the automatic annotation feature.
+
+* [`--annotation_output_dir`](https://github.com/googlegenomics/gcp-variant-transforms/blob/c4659bba2cf577d64f15db5cd9f477d9ea2b51b0/gcp_variant_transforms/options/variant_transform_options.py#L190)
+a path on Google Cloud Storage (GCS) where VEP output files are copied. The file
+hierarchy of input files is replicated at this location with the same file names
+followed by `_vep_output.vcf`.
+
+* [`--vep_image_uri`](https://github.com/googlegenomics/gcp-variant-transforms/blob/c4659bba2cf577d64f15db5cd9f477d9ea2b51b0/gcp_variant_transforms/options/variant_transform_options.py#L196)
+the docker image for VEP created using the
+[Dockerfile in variant-annotation](https://github.com/googlegenomics/variant-annotation/tree/master/batch/vep)
+GitHub repo.
+<!-- TODO(bashir2): Add the location of the image we maintain. -->
+
+* [`--vep_cache_path`](https://github.com/googlegenomics/gcp-variant-transforms/blob/c4659bba2cf577d64f15db5cd9f477d9ea2b51b0/gcp_variant_transforms/options/variant_transform_options.py#L200)
+the GCS location that has the compressed version of VEP cache. This file can be
+created using
+[build_vep_cache.sh](https://github.com/googlegenomics/variant-annotation/blob/master/batch/vep/build_vep_cache.sh)
+script.
+<!-- TODO(bashir2): Add the location of the cache we maintain. -->
+
+* [`--vep_info_field`](https://github.com/googlegenomics/gcp-variant-transforms/blob/c4659bba2cf577d64f15db5cd9f477d9ea2b51b0/gcp_variant_transforms/options/variant_transform_options.py#L204)
+by default, VT adds an INFO field called `CSQ_VT` to hold the new
+annotations; use this field to override the field name.
+
+* [`--vep_num_fork`](https://github.com/googlegenomics/gcp-variant-transforms/blob/c4659bba2cf577d64f15db5cd9f477d9ea2b51b0/gcp_variant_transforms/options/variant_transform_options.py#L208)
+number of forks when running VEP, see [`--fork` option of VEP](ensembl.org/info/docs/tools/vep/script/vep_options.html#opt_fork).
+
+For other parameters, like how many VMs to use or where the VMs should be
+located, the same parameters for
+[Dataflow pipeline execution](https://cloud.google.com/dataflow/pipelines/specifying-exec-params)
+are reused, e.g., `--num_workers`.
+
+## Sample Queries
+
+We imported the gnomAD genomes table using the features described in
+[Parsing Annotation Fields](#parsing-annotation-fields). Here is a sample
+query to extract high impact variants in the HBB gene which encodes the beta
+chain of Haemoglobin:
+```
+#standardSQL
+SELECT reference_name, start_position, reference_bases, ALT.alt, CSQ.*
+FROM vcf_imports_external.gnomad_genomes_chr_hg19 AS T, T.alternate_bases AS ALT, ALT.CSQ AS CSQ
+WHERE CSQ.SYMBOL = "HBB" AND CSQ.IMPACT = "HIGH"
+ORDER BY start_position
+```
+
+and here is an snapshot from BigQuery UI for this query:
+
+![A sample query on the gnomAD genomes table](images/gnomAD_HBB_query.png)
```

Comment:
nit: would be better to have the actual query inlined as text so that it's both easier to read and enables copy/paste.

Reply:
ah, nvm :) did not realize that the query is actually just above the image.